VogtPayroll4/Payroll.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace VogtPayroll4
+{
+    class Payroll
+    {
+        public List<Employee> employees;
+
+        public List<Employee> ReadEmployees()
+        {
+            EmployeePayrollConsoleReader EmployeePayrollConsoleReader = new EmployeePayrollConsoleReader();
+
+            return EmployeePayrollConsoleReader.CreateNewEmployee();
+        }
+
+        public void LoopThroughEmployeesList(List<Employee> employees)
```

Comment:
LoopThroughEmployeeList doesn't describe what this is doing. PrintAllEmployeeInfo() would be more descriptive.

You shouldn't need to pass in the list of employees. Really, you should have a `private List<Employee> _employees` on the class.

I'd go this route:
```c#
 class Payroll
    {
        private List<Employee> _employees = new List<Employee>();

        public void AddEmployees(List<Employee> employees)
        {
            _employees.AddRange(employees);
        }

        public void PrintAllEmployeeInfo() {
            for(var employee in _employees) {
                employee.PrintInfo();
            }
        }
```